tests/test-cases/log-level-validation.yaml

```diff
@@ -74,7 +74,7 @@ tests:
       stdout:
         - '^\n👽 Atmos (\d+\.\d+\.\d+|test) on [a-z]+/[a-z0-9_]+\n\n'
       stderr:
-        - "DEBU Found ENV var ATMOS_LOGS_LEVEL=Debug"
+        - "Set logs-level=debug logs-file=/dev/stderr"
```

Comment:
Do we have a test that tests the preference order of config, env, and flags?

Reply:
yes we already added that when we faced it previously. So we are good with that.